src/Compat.jl

```diff
@@ -199,6 +199,12 @@ if VERSION < v"1.4.0-DEV.551"
     Base.filter(f, t::Base.Any16) = Tuple(filter(f, collect(t)))
 end
 
+# https://github.com/JuliaLang/julia/pull/34652
+if VERSION < v"1.5.0-DEV.236"
```

Comment:
Could you double-check the version?
```
$ ./contrib/commit-name.sh 1297b518e3c6b616cbc8e398b2c4cc977ce7cafc
1.5.0-DEV.247
```
```suggestion
if VERSION < v"1.5.0-DEV.247"
```

Reply:
fixed in c8403f9